imports/ui/components/GeoSuggest.jsx

```diff
@@ -33,13 +33,23 @@ export class GeoSuggest extends React.Component {
   };
 
   render() {
+    /* -----------------------------------------------------------
+   These options will bias the autocomplete predictions toward 
+         Vancouver, Canada with a radius of 200 meters.
+	-------------------------------------------------------------*/
+    const searchOptions = {
+      location: new google.maps.LatLng(49.290338, -123.134112),
```

Comment:
Let's also center the map on Vancouver now, instead of Downtown :)

Reply:
Resolved